addons/traefik/1.7.x/tests/11-exec.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kudo.dev/v1beta1
+kind: TestStep
+commands:
+  - command: ./exec-test.sh
```

Comment:
I'm curious regarding the jump from index `02` to `11`

more curious regarding content of `exec-test.sh` -- what are you looking for?   perhaps it is a feature we can add to kuttl.   The testing of multiple contexts is an interesting use case.  authed, non-authed, mixed... 🤔 

The switch of server is unexpected tho... more 🤔

Reply:
after some thought.. I really concerned here... the switch of the kubectl context (with no return to previous state) will potentially challenge all future added tests or other tests in the testsuite.

Reply:
>I'm curious regarding the jump from index 02 to 11

Anybody that's used index-named files has come across the occasion where they had to renumber them all to insert one (sysvinit, for instance). I probably should have even left a bigger gap between 0 and 2.

> more curious regarding content of exec-test.sh -- what are you looking for? perhaps it is a feature we can add to kuttl. The testing of multiple contexts is an interesting use case. authed, non-authed, mixed... thinking

What this is for is testing that pipelining still works through the proxy. It was discovered that there was a deficiency in traefik (and possibly others) that was preventing pipeline connection upgrades (SPDY protocol in the current case).

This means that a connection from kubectl through the proxy to the api server needs to be made and needs to use one of the commands that causes the connection upgrade.

> the switch of the kubectl context (with no return to previous state) will potentially challenge all future added tests or other tests in the testsuite

Fair point. I'll change it to use it's own temporary file.